scalding-repl/src/main/scala/com/twitter/scalding/ReplImplicits.scala

```diff
@@ -58,12 +67,62 @@ trait BaseReplState {
     useHdfsMode_()
     customConfig -= mr1Key
     customConfig -= mr2Key
+    modeBanner
   }
 
   def useHdfsLocalMode() {
     useHdfsMode_()
     customConfig += mr1Key -> mrLocal
     customConfig += mr2Key -> mrLocal
+    modeBanner
+  }
+
+  def modeBanner: Unit = {
```

Comment:
also can we call this something like `printModeBanner` to make it clear what the side effect is?